clicommand/global.go

```diff
@@ -115,6 +115,14 @@ var (
 		EnvVar: "BUILDKITE_CONTAINER_ID",
 	}
 
+	KubernetesBootstrapConnectionTimeoutFlag = cli.IntFlag{
+		Name: "kubernetes-bootstrap-connection-timeout",
+		Usage: "This is intended to be used only by the Buildkite k8s stack " +
+			"(github.com/buildkite/agent-stack-k8s); it set the max time a container will wait " +
+			"to connect Agent.",
+		EnvVar: "BUILDKITE_KUBERNETES_BOOTSTRAP_CONNECTION_TIMEOUT",
+	}
+
```

Comment:
We've generally not put command-specific flags in `global.go`. `KubernetesContainerIDFlag` is currently shared with `bootstrap.go` and `kubernetes_bootstrap.go` which is why it's here. 

(At some point we should remove the legacy kubernetes-exec support from within bootstrap.go.)